doc/sslang-lrm.tex

```diff
@@ -715,7 +716,26 @@ \subsection{Sequences}
 
 
 \subsection{Pattern Matching}
-\fixme{todo}
+\begin{grammar}
+<exprBlk> ::= "match" <exprBlk> "{" <matchArms> "}"
+
+<matchArms> ::= matchArm "|" matchArms
```

Comment:
1. see comments above about casing and folding the `match_arms` inline
2. do something like `<match> := ...` to be consistent with other sections.

Reply:
Done